### PR TITLE
increase timeout for reboot, and streamline healthchecks

### PR DESCRIPTION
**Pull request by twaldrop**
increase timeout for reboots, delay healthchecks and retry less

### CODE REVIEW

File: playbooks/reboot_cluster.yml [hunk 1]
```diff
@@ -11,6 +11,8 @@
   - name: waiting for server to come back
     wait_for: host={{ inventory_hostname }}
               state=started
+              connect_timeout=900
```
Comment (nirajdp76) [inline playbooks/reboot_cluster.yml]: Can explain how connect_timeout and timeout works. What will we see on reboot with this changes?

**Comment by j2sol** [comment]
This needs to be rebased to pick up libvirt updates.

File: playbooks/reboot_cluster.yml (hunk 1 above)
Comment (twaldrop) [inline playbooks/reboot_cluster.yml]: Connect_timeout is the maximum amount of time to wait for the ssh connection, while timeout is the maximum amount of time to wait for the task to complete.  Looking at this again, I don't think connect_timeout actually needs to be this high.

Comment (twaldrop) [inline playbooks/reboot_cluster.yml]: Reduced connect_timeout to 60 seconds

**Pull request merged**